articles/quickstarts/install-jupyter.md

```diff
@@ -14,18 +14,35 @@ Install the QDK for developing Q# operations on Q# Jupyter Notebooks.
 
 Jupyter Notebooks allow in-place code execution alongside instructions, notes, and other content. This environment is ideal for writing Q# code with embedded explanations or quantum computing interactive tutorials. Here's what you need to do to start creating your own Q# notebooks.
 
+> [!NOTE]
+> * In Q# Jupyter Notebooks, you can only run Q# code, and the operations cannot be called from external host programs (e.g. Python or C# files). This environment is not appropriate if your goal is to combine an external classical host program with the quantum program.
+
+## Install the IQ# Jupyter kernel
+
 IQ# (pronounced i-q-sharp) is an extension primarily used by Jupyter and Python to the .NET Core SDK that provides the core functionality for compiling and simulating Q# operations.
 
-> [!NOTE]
-> * In Q# Jupyter Notebooks you can only run Q# code, and the operations cannot be called from external host programs (e.g. Python or C# files). This environment is not appropriate if your goal is to combine an external classical host program with the quantum program.
+### [Install using conda (recommended)](#tab/tabid-conda)
+
+1. Prerequisites:
+
+    - Install [Miniconda](https://docs.conda.io/en/latest/miniconda.html) or [Anaconda](https://www.anaconda.com/products/individual#Downloads).
+    - (optional) Create a new conda environment by running `conda create --name qsharp-env`. Activate this environment by running `conda activate qsharp-env`.
+
+1. Install the `qsharp` package in your preferred conda environment:
 
-1. Prerequisites
+    ```
+    conda install -c quantum-engineering qsharp
+    ```
```

Comment:
Yes. In fact, neither `iqsharp` nor `qsharp` depends on `notebook` (which I think makes sense technically, since it's not a requirement of the package). I'll add `notebook` to the command line here, so that people get Jupyter Notebook when following these steps.